apps/api/src/coscrad-cli/ingest-media-items.cli-command.ts

```diff
@@ -67,10 +73,36 @@ export class IngestMediaItemsCliCommand extends CliCommandRunner {
 
         const generatedIds = await this.idManager.generateMany(partialPayloads.length);
 
+        const mediaLengthMap = new Map<string, number>();
+
+        for (const { filename, mimeType } of partialPayloads) {
+            const ffprobeResult = await ffprobe(`${directory}/${filename}`);
+
+            const { streams } = ffprobeResult;
+
+            const duration = streams?.[0].duration;
```

Comment:
In the future, we will want other properties as well, for example, image dimensions.

Reply:
Add a story for supporting this.

Reply:
[Here](https://www.pivotaltracker.com/story/show/186713537) is the story.